handbook/recruitment/overview.md

```diff
@@ -38,3 +38,52 @@ To provide support to our less experienced members and ensure a productive envir
+
+A few things to note:
+
+* Make sure to follow the [steps for a social media campaign](/handbook/tools/social-media.md#social-media-campaigns) as guidance, especially regarding setting up the Google Drive folder correctly, to make sure we have reference material in the future.
+* Do not use the word "hiring", as we do not provide pay - "recruitment" is a more accurate and less intimidating term.
+
+### Applications
+
+All materials pertaining to accepting applications (Google Forms, spreadsheets, and so on) should be tracked in the [Recruitment folder (under Leads)](https://drive.google.com/drive/u/0/folders/0BwdNv1PZjDeXNXZiQm55Q2VBc1U).
+
+#### Accepting applications
+
+Applications are generally accepted through Google Form submissions.
+
+1. Create a folder under Recruitment for the recruitment season (for example, `YYYY MM`).
+2. Make a copy of the template forms provided in the Recruitment folder and make the appropriate adjustments relevant to the recruitment season.
+3. Enable responses under the "Responses" tab of the form.
+4. Make the appropriate website updates:
+   1. [`docs.ubclaunchpad.com`](https://github.com/ubclaunchpad/docs): make sure our [role description pages](/handbook/README.md#recruitment) are up to date, with links to the relevant Google Forms at the bottom of each page.
+   2. [`ubclaunchpad.com`](https://github.com/ubclaunchpad/ubclaunchpad.com): [configure recruitment status](https://ubclaunchpad.com/config/#configuring-recruitment-status).
+
+#### Closing applications
+
+1. Disable responses under the "Responses" tab of the form.
+2. Make the appropriate website updates:
+   1. [`docs.ubclaunchpad.com`](https://github.com/ubclaunchpad/docs): remove links to application Google Forms from the [role description pages](/handbook/README.md#recruitment)
+   2. [`ubclaunchpad.com`](https://github.com/ubclaunchpad/ubclaunchpad.com): [configure recruitment status](https://ubclaunchpad.com/config/#configuring-recruitment-status).
+
+#### Screening applicants
+
+To screen applications, we provide a set of criteria, each to be graded on a 0-2 scale from "unsatisfactory" to "excellent" *relative to the applicant's skill level* so that we can try to achieve the experience distributions described in our [recruitment goals](#goals). These scores are used to determine which candidates we interview.
+
+For most of our entry roles (developers, designers, and strategy), we have the following criteria:
+
+* C1: Willingness to learn (self-drive, taking time outside of classes and work to learn and build things)
+* C2: Passion and interest (for example, comprehensiveness of answer to "tell us about a project", even if not technically comprehensive)
+* C3: Rate the best of the provided items from the OR/AND's from the relevant skill bucket they fall in
```

Comment:
I wonder if there is a clearer way to describe this one

Reply:
🤔 yeah it's quite a mouthful, but seems to do the job after you turn it around in your head for a bit 😅 

Maybe a separate PR to better define the conditions in the goals and giving them a better name and/or format than AND/OR?